### DISCUSSION TITLE
Conditions - Chain multiple conditions in IF statement

### DISCUSSION THREAD

**Discussion by kaip-e** [maizzle/framework#915] [General]
Hey. :)
I am using maizzle for a short time now and really love the framework so far! (Super fast and easytemplating with tailwind)

However, I stumbled across some cases where I need to chain multiple conditions within one if (else if) and can not figure out, how to properly do this.

Let's say I got a component.html using another subcomponent. This subcomponent is dynamic and used in different components. By passing a 'type' to this component i want to render different tables / styles. How can I check for multiple conditions within the subcomponent using the || or && operator?

 **component.html referring to my subcomponent**
`<component type="approval" src="subComponent.html"/>`

**Condition within my subcomponent does not work**
`<if condition="type !== 'paymentInAdvance' || 'approval'">
     [...]
</if>`

**Comment by cossssmin** [comment]
Hi, what Maizzle framework version are you using? (`maizzle -v` to find out).

The condition should be written as `type !== 'paymentInAdvance' && type !== 'approval'`, which makes sure `type` is neither of those.

I think this should also work:

```html
<if condition="!['paymentInAdvance', 'approval'].includes(type)">
  ...
</if>
```

Finally, you could try a [`<switch>`](https://maizzle.com/docs/tags#switch):

```html
<switch expression="type">
  <case n="'paymentInAdvance'">
    ...
  </case>
  <case n="'approval'">
    ...
  </case>
  <default>
    ...
  </default>
</switch>
```

**Reply by kaip-e** [reply]
Thanks for the quick answer! This did fix the issue I had. :)
